techniques/failures/F2.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Failure of Success Criterion 1.3.1 due to using changes in text presentation to convey information without using the appropriate markup or text</title><link rel="stylesheet" type="text/css" href="../../css/editors.css" class="remove"/></head><body><h1>Failure of Success Criterion 1.3.1 due to using changes in text presentation to convey information without using the appropriate markup or text</h1><section class="meta"><p class="id">ID: F2</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
       <p>All technologies that support images or presentation markup.</p>
    </section><section id="description"><h2>Description</h2>
-      <p>This document describes a failure that occurs when a change in the appearance of text conveys meaning without using appropriate semantic markup. This failure also applies to images of text that are not enclosed in the appropriate semantic markup.</p>
+      <p>This document describes a failure that occurs when a change in the appearance of text conveys meaning without using appropriate semantic markup, or without using text styling that is conveyed by assistive technology. This failure also applies to images of text that are not enclosed in the appropriate semantic markup.</p>
```

Comment:
(per TF discussion) It feels off to add this clause when the rest of the PR is specifically *removing* the only example where it would apply. In the headings examples, it wouldn't be sufficient for ATs to convey a change in text styling; that wouldn't be sufficient to convey the semantic heading structure.

I would lean more towards updating the technique to more specifically talk about not using non-semantic visual techniques to present stuff that looks semantically relevant (the 2 remaining examples with headings are good, could also add something like "using a left-margin and images instead of a semantic list to indicate a list" if we want a non-heading example)

Reply:
okay, thanks @dbjorge.

I will pull back on additive material for now.

Reply:
@dbjorge please re-review so this can be merged if/when supported by WG.